proj/src/main/webapp/script.js

```diff
@@ -344,12 +353,10 @@ function makeChatPreview(name, messageObj, tag, chatId) {
 }
 
 function populateSidebar() {
-    // Initialize auth object
-    const auth = firebase.auth();
-
+    // const auth = firebase.auth();
     // const iud = auth.currentUser.uid;
     const iud = "testuserwithdict";
```

Comment:
Is this supposed to be "uid"?  "IUD" generally means something rather different.

Reply:
yep... it seems i've made a very unfortunate spelling error